flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
Any reason to limit it to x86_64?

Reply:
as of now because it is not tested extensively